editor/src/components/canvas/canvas-strategies/strategies/absolute-duplicate-strategy.tsx

```diff
@@ -76,36 +78,57 @@ export function absoluteDuplicateStrategy(
           ...customStrategyState.duplicatedElementNewUids,
         }
         let duplicateCommands: Array<CanvasCommand> = []
-        let newPaths: Array<ElementPath> = []
 
         flattenedSelectionForMultiSelect.forEach((selectedElement) => {
           const selectedElementString = EP.toString(selectedElement)
           const newUid =
             duplicatedElementNewUids[selectedElementString] ??
             generateUidWithExistingComponents(canvasState.projectContents)
-          duplicatedElementNewUids[selectedElementString] = newUid
-
-          const newPath = EP.appendToPath(EP.parentPath(selectedElement), newUid)
 
-          newPaths.push(newPath)
+          duplicatedElementNewUids[selectedElementString] = newUid
           duplicateCommands.push(duplicateElement('always', selectedElement, newUid, 'before'))
         })
 
         return strategyApplicationResult(
           [
             ...duplicateCommands,
-            setElementsToRerenderCommand([...selectedElements, ...newPaths]),
-            updateSelectedViews('always', selectedElements),
-            updateFunctionCommand('always', (editorState, commandLifecycle) =>
-              runMoveStrategy(
-                canvasState,
-                editorState,
+            updateFunctionCommand('always', (state, commandLifecycle) => {
+              const selectedUids = selectedElements.map(EP.toUid)
+              const newParentPath = EP.parentPath(
+                Object.values(state.canvas.controls.reparentedToPaths)[0],
```

Comment:
I'd recommend making this indexing safe, as it's assuming there's a value in that object.

Reply:
added it in https://github.com/concrete-utopia/utopia/pull/3917/commits/d091d0526c484342a7b5ee4b37499b63603c0ef8